src/index.js

```diff
@@ -344,6 +352,10 @@ module.exports = S => {
               const contentType = request.mime || defaultContentType;
               const requestTemplate = requestTemplates[contentType];
 
+              if (contentType === 'application/json') {
+                request.payload = JSON.parse(request.payload);
```

Comment:
this breaks my code.

when my method = OPTIONS.  my payload: <Buffer > seems to be an empty string and trying to parse it... blows up

Debug: internal, implementation, error 
    SyntaxError: Uncaught error: Unexpected end of input
  at Object.parse (native)